src/BlockTypes/CollectionPriceFilter.php

```diff
@@ -16,6 +16,65 @@ final class CollectionPriceFilter extends AbstractBlock {
 	const MIN_PRICE_QUERY_VAR = 'min_price';
 	const MAX_PRICE_QUERY_VAR = 'max_price';
 
+	/**
+	 * Initialize this block type.
+	 *
+	 * - Hook into WP lifecycle.
+	 * - Register the block with WordPress.
+	 */
+	protected function initialize() {
+		parent::initialize();
+
+		add_filter( 'collection_active_filters_data', array( $this, 'register_active_filters_data' ), 10, 2 );
+	}
+
+	/**
+	 * Register the active filters data.
+	 *
+	 * @param array $data   The active filters data.
+	 * @param array $params The query param parsed from the URL.
+	 * @return array Active filters data.
+	 */
+	public function register_active_filters_data( $data, $params ) {
+		$min_price           = intval( $params[ self::MIN_PRICE_QUERY_VAR ] ?? 0 );
+		$max_price           = intval( $params[ self::MAX_PRICE_QUERY_VAR ] ?? 0 );
+		$formatted_min_price = $min_price ? wc_price( $min_price, array( 'decimals' => 0 ) ) : null;
+		$formatted_max_price = $max_price ? wc_price( $max_price, array( 'decimals' => 0 ) ) : null;
+
+		if ( ! $formatted_min_price && ! $formatted_max_price ) {
+			return $data;
+		}
+
+		if ( $formatted_min_price && $formatted_max_price ) {
+			/* translators: %1$s and %2$s are the formatted minimum and maximum prices respectively. */
+			$title = sprintf( __( 'Between %1$s and %2$s', 'woo-gutenberg-products-block' ), $formatted_min_price, $formatted_max_price );
```

Comment:
It looks like a styling issue causes the spans to swallow up the whitespace between the word " and "

<img width="1179" alt="Screenshot 2023-12-01 at 2 01 10 PM" src="https://github.com/woocommerce/woocommerce-blocks/assets/1281828/beb6c96e-ffa0-4135-b9e1-d07455f67092">

Reply:
I fixed this one by removing the HTML tag in the price, I think it's sufficient enough in that case.